Makefile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 # Various versions - Minishift, default OpenShift, default B2D ISO
-MINISHIFT_VERSION = 1.1.0
+MINISHIFT_VERSION = 1.2.0
```

Comment:
@LalatenduMohanty suggested to also integrate @budhrg's PR. At the moment I am waiting for this...

When submitted as PR, we also have a build and integration triggered to confirm this build is OK.